src/Components/FeelingsLog/FeelingsLog.js

```diff
@@ -1,10 +1,45 @@
-import React from "react";
+import React from 'react';
+import './FeelingsLog.css';
+import { DateTime } from 'luxon'
 
-const FeelingsLog = () => {
+const FeelingsLog = ({logs}) => {
+
+  const toTitleCase = (item) => {
+    return `${ item.slice(0, 1).toUpperCase() }${ item.slice(1) }`;
+  }
+
+  const sorted = logs
+    .map(log => ({
+      ...log,
+      date: DateTime.fromFormat(log.date, 'yyyy-MM-dd'),
+    }))
+    .sort((a, b) => b.date - a.date);
+
+  const  userLogs = sorted.map(log => {
+    return (<tr key={log.entryId}>
+        <td>{log.date.toLocaleString(DateTime.DATE_MED)}</td>
+        <td>{toTitleCase(log.feeling)}</td>
+        <td>{log.activity}</td>
+        <td>{log.journalBool ? '📓' : null}</td>
+      </tr>)
+  })
   return (
-    <>
-    <h2>Feelings Log</h2>
-    </>
+    <section className='feelingsLog'>
+      <table>
+        <thead>
+          <tr>
+            <th>Date</th>
+            <th>Feeling</th>
+            <th>Activity</th>
+            <th>Journal</th>
+          </tr>
+        </thead>
+        <tbody>
+        {userLogs}
+        </tbody>
+      </table>
+
+    </section>
```

Comment:
YES - I love bootstrap. Wouldn't mind using their package tbh! Great call here :) The table looks super clean.

Reply:
oh REALLY?? I should have mentioned that, i didn't wanna just implement it without chatting with y'all. It would not have been ideal for me to just use it because if some of the class names were the same as bootstraps (in any file) those styles would have overridden them!